Domains/3-DisciplineOther/StructuralAnalysis/StructuralComponents.md

```diff
@@ -0,0 +1,42 @@
+# Structural Components Hierrarchy
```

Comment:
The content of this particular file would need to move to the remarks of the StructuralAnalysis, otherwise it will not be included in the final documentation of the schema. See https://github.com/iTwin/bis-schemas/blob/master/docs/schema-documentation-style-guide.md for more details.

Another option is to include it in the documentation for all "Structural schemas", either here:
https://www.itwinjs.org/bis/domains/discipline-physical-domains/structural-domains/
or here:
https://www.itwinjs.org/bis/domains/discipline-other-domains/structural-domains/

... but those two pages are in the itwinjs-core repo, not in the bis-schemas repo.

Reply:
Would it be fine to skip this part in this PR and resolve it later?
The placement of documentation would not affect how the schema is implemented

Reply:
I've moved the contents of StructuralComponents.md to Profiles remarks file